server/package.json

```diff
@@ -58,6 +58,7 @@
     "cookie-parser": "^1.4.6",
     "diskusage": "^1.1.3",
     "exiftool-vendored": "^19.0.0",
+    "exiftool-vendored.pl": "^12.54.0",
```

Comment:
I hope you are aware that this is not meant to be used directly as a dep?

"You shouldn't include either the exiftool-vendored.exe or exiftool-vendored.pl as direct dependencies to your project." - https://www.npmjs.com/package/exiftool-vendored install section

Reply:
Yeah, we looked at this. That works fine, but we are specifically running `npm ci --omit=optional` to decrease the number of unused packages that are installed on the docker image. Unfortunately this is marked as an optional and there really isn't a way around getting it installed without listing it as a dependency or installing all our optional deps.